README.md

```diff
@@ -1,331 +1,27 @@
-
-##### Errors related to _too many open files_
-You may receive connection errors similar to the following:
-
-`Failed to open TCP connection to 127.0.0.1:9515 (Too many open files - socket(2) for "127.0.0.1" port 9515)`
-
-You are encountering you OS's [limits on allowed file descriptors](https://wilsonmar.github.io/maximum-limits/). Check the limits with both:
-* `ulimit -n`
-* `launchctl limit maxfiles`
-
-Try this to increase the user limit:
-```
-$ ulimit -Sn 65536 && make test
-```
-To set this _permanently_, add the following to your `~/.zshrc` or `~/.bash_profile` file, depending on your shell:
-```
-ulimit -Sn 65536
-```
-
-If you are running MacOS, you may find it is not taking your revised ulimit seriously. [You must insist.](https://medium.com/mindful-technology/too-many-open-files-limit-ulimit-on-mac-os-x-add0f1bfddde) Run this command to edit a property list file:
-```
-sudo nano /Library/LaunchDaemons/limit.maxfiles.plist
-```
-Paste the following contents into the text editor:
-```
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
-          "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-  <dict>
-    <key>Label</key>
-    <string>limit.maxfiles</string>
-    <key>ProgramArguments</key>
-    <array>
-      <string>launchctl</string>
-      <string>limit</string>
-      <string>maxfiles</string>
-      <string>524288</string>
-      <string>524288</string>
-    </array>
-    <key>RunAtLoad</key>
-    <true/>
-    <key>ServiceIPC</key>
-    <false/>
-  </dict>
-</plist>
-
-```
-Use Control+X to save the file.
-
-Restart your Mac to cause the .plist to take effect. Check the limits again and you should see both `ulimit -n` and `launchctl limit maxfiles` return a limit of 524288.
-
-##### Errors related to _sassc_
-
-If you are getting the error:
-```
-LoadError: cannot load such file -- sassc
-```
-Try `make run` for a short time, then use Ctrl+C to kill it
-
-##### Errors relating to OpenSSL versions
+- [The Contributing Guide](CONTRIBUTING.md) includes basic guidelines around pull requests, commit messages, and the code review process.
+- [The Login.gov Handbook](https://handbook.login.gov/) describes organizational practices, including process runbooks and team structures.
 
-If you get this error during test runs:
-```
-     Failure/Error: JWT::JWK.import(certs_response[:keys].first).public_key
-     OpenSSL::PKey::PKeyError:
-       rsa#set_key= is incompatible with OpenSSL 3.0
-```
-See [this document](docs/FixingOpenSSLVersionProblem.md) for how to fix it.
+## Documentation
 
+- [Back-end Architecture](docs/backend.md)
+- [Docker](docs/Docker.md)
+- [Front-end Architecture](docs/frontend.md)
+- [How to fix a recurrent problem with OpenSSL and some of our tests](docs/FixingOpenSSLVersionProblem.md)
```

Comment:
Maybe we merge this into the new Troubleshooting documentation?

Reply:
updated in 908222796c281c37f52d043840560aeb46bad841